test/workflow-ui/job.ts

```diff
@@ -75,6 +75,10 @@ describe('Workflow UI job route', function () {
           const listing = this.res.text;
           expect(listing).to.contain(mustache.render('<a href="/workflow-ui">Jobs</a>', {}));
         });
+        it('sets the page limit input to the expected value', function () {
+          const listing = this.res.text;
+          expect(listing).to.contain('<input name="limit" type="number" class="form-control" value="1">');
```

Comment:
We should add some tests with values larger than the max allowed size (which looks to be 2000 based on testing).

Reply:
Added the test.